cha/implementaion/playback/meta.tex

```diff
@@ -0,0 +1,28 @@
+In this chapter we explore some key elements in the development of our application related to the playback of audio data.
+The chapter includes our decisions, the reasoning behind them, some of the complications we encountered and how we dealt with them.
+Before we present the implementation and what that entails, we firstly present some audio data terminology, as well as the Android sound stack, as this domain knowledge is used in our implementation.
+
+\begin{description}
+    \item [Sample Rate]\cite{sample_rate_std} \hfill \\
+    Sample rate is the number of audio samples per channel per second.
+    The rate may vary from file to file with $44,100\ Hz$ being the de facto and most supported sampling rate.
+    \item [Bit Depth]\cite{bit_depth} \hfill \\
+    Bit depth is largely dependant on the data encoding used, and if relevant also described alongside sample rate, e.g.\ a data rate of 44.1 kHz/16 translates to a sample rate of 44,100 and a bit depth of 16.
+    Bit depth is the number of bits used per sample.
+    In our project we work with MP3 and PCM encoding, and as such we are only concerned with these two formats.
+    MP3 does not encode any bit depth in its data, as such the bit depth for the \ac{PCM} data depends on the decoder settings.
```

Comment:
I don't actually think this makes any sense. MP3 stores data in the frequency domain in a huffman coded model. Talking about sampling rate and bitdepth doesn't make any sense in a format like that. What is true is that the "resampling" of an MP3 file can be done at any bitdepth. That means that mpg123 can decode the frequency domain mp3 data into 16bit 8bit or float time domain data.

It's pretty complex and i don't understand all of it. This paper has a nice diagram: https://www.mp3-tech.org/programmer/docs/mp3_theory.pdf